precompile/allow_list.go

```diff
@@ -153,7 +154,7 @@ func createAllowListRoleSetter(precompileAddr common.Address, role AllowListRole
 		modifyAddress := common.BytesToAddress(input)
 
 		if readOnly {
-			return nil, remainingGas, ErrWriteProtection
+			return nil, remainingGas, vmerrs.ErrWriteProtection
```

Comment:
ooh seems I imported `go-ethereum` vm package, instead of subnet-evm. that's why it worked haha

Reply:
but certainly not a bad idea? :P

Reply:
I think we should migrate to a separate package. We should not rely on any geth packages where we also have the same package present in our code.